content/purgatorio/now-go-build/transfer-learning.md

```diff
@@ -1,5 +1,111 @@
 ---
 title: Transfer Learning
 author: clone95
-description: ---
----
\ No newline at end of file
+description: A detailed guide about what is Transfer Learning, how to use it, and a lot of resources found on the web that shares pre-trained models.
+---
+
+# Index 
+
+- [What is Transfer Learning](#Why-Do-You-Need-Transfe-Learning)
+- [Transfer Learning Magical Properties](#[TransferLearning-Inner-Workings)
+- [How Does Transfer Learning Work](#How-Does-Transfer-Learning-Work)
+- [Where to Find Pretrained Models](#Where-to-Find-Pretrained-Models)
+- [Conclusions](#Conclusions)
+
+##  What is Transfer Learning
+
+[Transfer Learning](https://en.wikipedia.org/wiki/Transfer_learning) is a research problem in Machine Learning that focuses on storing knowledge gained while solving one problem and applying it to a different but related problem. 
+
+This is done by _pre-training_ a model on a dataset (usually bigger and with generic content, like [ImageNet](http://www.image-net.org/)), and then _fine-tuning_ the model on a smaller and similar dataset.
```

Comment:
> This is done by _pre-training_ a model

This could have read better if you said "This is done by using an existing _pre-trained_  model..."

Otherwise to a first-time it sounds like oh I have to first pretrain a model and then use it to transfer learn - that is a redundant step and defeats the purpose of transfer learning.